share/spice/sales_tax_holiday/sales_tax_holiday.js

```diff
@@ -0,0 +1,70 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_sales_tax_holiday = function(api_result){
+        // Error checking
+        var errorMessage = "UNKNOWN ERROR";
+        var isError = false;
+        if (!api_result 
+         || (typeof api_result.header === 'undefined')
+         || api_result.header.status !== "SUCCESS" 
+         || !api_result.taxHolidays
+         || (typeof api_result.taxHolidays === 'undefined')
+         || api_result.taxHolidays.length < 1) {
+            isError = true;
+            if (api_result.header && api_result.header.errorMessage) {
+                errorMessage = api_result.header.errorMessage;
+            } else {         
+              return Spice.failed('sales_tax_holiday');
+            }
+        }
+
+        // Display
+        Spice.add({
+            id: "snapcx_sales_tax_holiday",
+            name: "Finance",
+            data: isError? {"errorMessage" : errorMessage} : api_result.taxHolidays,
+            meta: {
+                sourceName: "snapCX.io",
```

Comment:
We should only need `sourceName: "SnapCX", sourceUrl: "https://snapcx.io"` -- if you have a favicon we'll pick it up.

Do you have an endpoint on your website we can send the user to that will provide more info about their specific search? The link is intended to show them more detailed info about the IA they're looking at. Going to a landing page isn't a great user experience.